packages/core/src/__tests__/methods/identify.test.ts

```diff
@@ -77,7 +77,7 @@ describe('methods #identify', () => {
     });
   });
 
-  it('does not persist identity traits accross events', async () => {
+  it.only('does not persist identity traits accross events', async () => {
```

Comment:
Looks like we left a `.only` here.

Reply:
This is exactly why I need a code review for these big changes!